cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -58,6 +71,7 @@ import Prelude ()
 import Prelude.Compat
 import Text.URI
 
+-- cryptol package:
```

Comment:
For consistency with the [conventions in the `saw-script` package](https://github.com/GaloisInc/saw-script/blob/d8e2b6d4b491de31f22d89e4850ea519c37eb837/src/SAWScript/Crucible/LLVM/Builtins.hs#L140), let's shorten this to:

```suggestion
-- cryptol
```